packages/flutter/lib/material.dart

```diff
@@ -6,9 +6,12 @@
 ///
 /// To use, import `package:flutter/material.dart`.
 ///
-/// See
-/// [material.google.com](https://material.google.com/)
-/// for an introduction to Material Design.
+/// See also:
+///
+///  * [material.google.com](https://material.google.com/)
```

Comment:
I'd recommend swapping the order of these links. The /widgets should be much more useful for a Flutter developer.